debian/postinst

```diff
@@ -117,6 +120,12 @@ case "$1" in
             echo "org.jitsi.videobridge.xmpp.user.shard.MUC_NICKNAME=$(uuidgen)" >> $NEW_JITSI_CONFIG
         fi
 
+        if [ ! -f $HOCON_CONFIG ]; then
+            echo "Generating an empty hocon config"
+            echo "videobridge {
+            }" >> $HOCON_CONFIG
```

Comment:
If the file doesn't exist you don't need `>>`, just `>`, though I suppose it doesn't matter.

Should the file have a comment at the top giving a ref to the appropriate point in the JVB docs for its contents and syntax?

Reply:
I think it'll be the other way around: docs will point to this file.